aiida/cmdline/commands/cmd_data/cmd_structure.py

```diff
@@ -64,7 +64,7 @@ def structure():
 @list_options
 @decorators.with_dbenv()
 def structure_list(elements, raw, formula_mode, past_days, groups, all_users):
-    """List stored StructureData objects."""
+    """List StructureData objects stored in the database."""
```

Comment:
stored in the database is a bit superfluous, no? I mean what else is it going to show? You don't say this phrase anywhere else either even though it also applies there